db/migrate/20180605210436_add_title_cves_to_openscap_rule_results.rb

```diff
@@ -0,0 +1,6 @@
+class AddTitleCvesToOpenscapRuleResults < ActiveRecord::Migration[5.0]
+  def change
+    add_column :openscap_rule_results, :title, :string
+    add_column :openscap_rule_results, :cves, :string
```

Comment:
@hsong-rh Just checking. Can there be more than one `cves` per record?

Reply:
No, it should include all CVE references by appending them.

Reply:
It's delimited by commas.